src/components/button/code/less/_button.less

```diff
@@ -63,7 +63,7 @@
 .button-variant(@background; @border) {
   background-color: @background;
       border-color: @border;
-             color: contrast(@background, #414141, #ffffff, 60%);
+             color: contrast(@background, #414141, #ffffff, 70%);
```

Comment:
Definitely! if this comes up more than once ill move it out into it's over variable or mixin